cypress/support/config/services.js

```diff
@@ -893,7 +893,7 @@ const services = {
         path:
           Cypress.env('APP_ENV') === 'live' || Cypress.env('APP_ENV') === 'test'
```

Comment:
Does this test (any many others with assets now on `test`) still need to be prevented from running on `test`? I'm more curious than anything.
Cc @ryanmccombe

Reply:
Do you mean it needs to be prevented from running on `live`?

Good point - I think you're right - this asset does not exist on live

But neither did the previous one, so I'm not too clear why this test ever worked - will investigate!

Reply:
Good point - I think you're right

This asset is on TEST, so we could enable these E2Es on `TEST` again